index-clip.md

```diff
@@ -0,0 +1,114 @@
+
+### 3.1 From scratch for de-multiplexing FASTQ files
+
+If your data is not de-multiplexed yet, then use the following workflows. The user has to provide the in-line barcodes in a tab-delimited tabular format, for example: </br>
+rep1  TTAG </br>
+rep2	TGGC </br>
+rep3	TTAA </br>
+The raw data needs to be in FASTA or FASTQ format as a list of dataset pairs.
+
+- <a href="https://github.com/Florian-H-Lab/CLIP-Explorer/blob/master/eCLIP/1_Demultiplex_eCLIP.ga">Workflow to de-multiplex eCLIP read library</a>
+- <a href="https://github.com/Florian-H-Lab/CLIP-Explorer/blob/master/iCLIP/2_Demultiplex_iCLIP.ga">Workflow to de-multiplex iCLIP read library</a>
+
+
+### 3.2 From scratch with de-multiplexed FASTQ files
+
+You can choose between three different types of peak calling for the data analysis of eCLIP and iCLIP data. The data specification of each of the peak calling algorithms is listed below:
+
+**Table 1**: Data specification of the different peak calling algorithms.
+
+| Tool | Biological Replicates (Yes/No) | Control Data (Yes/No) |
+| ---            | :-:      | :-:     |
+| <a href="https://github.com/tbischler/PEAKachu">PEAKachu</a>            | Yes | Yes   |
+| <a href="https://doi.org/10.1186/s13059-017-1364-2">PureCLIP</a>            | No | Yes   |
+| <a href="https://doi.org/10.1093/bioinformatics/bts569">Piranha</a>           | No | No   |
+
+#### Note if you have used the de-mutliplexing workflows:
+If you used the preceding workflows for de-multiplexing, then remove the steps of `Cutadapt` and `UMI-tools extract` from the following workflows to analyse your data. Simply, import the workflow into you account, remove the tools and connect the lose end directly to the alignment step.
+
+#### Note if you use eCLIP data of Nostrand et al. (2016):
+The workflow for the eCLIP data of [Nostrand et al. (2016)](https://doi.org/10.1038/nmeth.3810) was used to analyse the data of RBFOX2. Beware when using other data of the study of [Nostrand et al. (2016)](https://doi.org/10.1038/nmeth.3810), because the size of the unique molecular identifier (UMI) can be different. The workflow is set to a UMI of five nucleotides. You can change this by importing the workflow into your account and amend the parameter `Cut bases from reads before adapter trimming` of the second `Cutadapt` step for the CLIP and control data.
+
+#### eCLIP
+- <a href="https://github.com/Florian-H-Lab/CLIP-Explorer/blob/master/eCLIP/1_CLIPseq-Explorer_demulti_PEAKachu_Nostrand_eCLIP_hg19_N5.ga">Workflow for the eCLIP data of Nostrand et al. (2016)</a>
```

Comment:
Should those be links to Galaxy workflows in Galaxy and not github? I guess we can change this later, as soon as the subdomain is in place.